dataset_examples/ACAV.md

```diff
@@ -0,0 +1,36 @@
+# [ACAV100M](https://acav100m.github.io)
+ACAV100M is a dataset of 100M videos with high audio-visual correspondance.
+
+## Download the metdata
```

Comment:
Would be great to either use the original data, either provide a script doing this preprocessing

Can we also add a `wget` command to download the original file ?

This may seems trivial but it greatly speed up the process of downloading several datasets to not have to dig this up every time

Reply:
Yeah I think I want to just use the original data by adding the functionality to extract YouTube metadata during processing.